subsys/metrics/src/main/resources/default-metrics.conf

```diff
@@ -2,6 +2,21 @@
 enabled = false
 reporter.enabled = true
 
+# Enable Galley to measure artifact downloading time. Default false.
+#
+#measure.transport = true
+
+# Specify a repository list for those to measure artifact downloading time. This works only if
+# measure.transport is true.
+#
+# Format: comma split item of "<package>:<type>:<name>"
+# The default "<package>:<type>" is "maven:remote" if ignored. You may use wildcard '*' so those would be measured
+# as a whole, e.g., koji-*
+#
+# Example: measure.repos = maven:remote:test,public,koji-*
+#
+#measure.transport.repos = public
```

Comment:
another change for the default example to 'central'

Reply:
Ok